### PR TITLE
add bold to bolded bit of locale string

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
So it matches this: https://cloud.githubusercontent.com/assets/12228435/10294966/369c8ede-6b72-11e5-9dd5-76e66e00092c.png

**Comment by alicoding** [comment]
Going to wait for your update on this patch for the new line...

**Comment by ScottDowne** [comment]
@alicoding updated.

File: less/shared.less [hunk 1]
```diff
@@ -88,8 +88,10 @@ img {
   height: auto;
 }
 
-br {
-  display: none;
+@media screen and (max-width: 480px) {
```
Comment (alicoding) [inline less/shared.less]: Good call!

**Comment by ScottDowne** [comment]
Ugh, messed it up

**Comment by ScottDowne** [comment]
Ok, good to go now.

**Pull request merged**